src/reference.yaml

```diff
@@ -14902,164 +14712,7 @@ paths:
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-        '400':
-          description: Client error.
-          x-errors:
-            - code: vans_not_found
-              category: base
-              message: VANs were not found
-              params: ''
-            - code: scope_not_in_range
-              category: scope
-              message: "scope should be in range: own, all, clients"
-              params: '{ "range" => "own, all, clients" }'
-          schema:
-            $ref: '#/definitions/GetVANsError'
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-        '401':
-          description: Unauthorized.
-          x-errors:
-            - code: invalid_supplied_credentials
-              category: username
-              message: Authentication failed with the supplied credentials
-              params: ''
-          schema:
-            $ref: '#/definitions/UnauthorizedError'
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-        '404':
-          description: Resource not found.
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-        '429':
-          description: Too many requests.
-          x-errors:
-            - code: too_many_requests
-              category: base
-              message: >-
-                Too many requests have been made to the api. Please refer to the
-                Developer Center for more information
-              params: ''
-          schema:
-            $ref: '#/definitions/RateLimitError'
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-        '500':
-          description: Internal server error
-          x-errors:
-            - code: internal_server_error
-              category: base
-              message: Internal server error
-              params: ''
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-        '503':
-          description: Service is temporary unavailable
-          x-errors:
-            - code: service_unavailable
-              category: base
-              message: Service is temporary unavailable
-              params: ''
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-        default:
-          description: Unexpected error.
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-  /withdrawal_accounts/find:
+    /withdrawal_accounts/find:
```

Comment:
This resource is now mis-aligned. delete the additional spaces

Reply:
Thanks - that's sorted it 👍